### DISCUSSION TITLE
Trouble with ParameterStudy

### DISCUSSION THREAD

**Discussion by cameronmcelfresh** [idaholab/moose#26126] [Q&A General] [answered]
Hi, I've been having issues getting the "ParameterStudy" to run a simple parameter study using the Heat Conduction module. I have been able to successfully run the test cases provided in the Stochastic Tools directory and my input file works using the Heat Conduction Module.

The error that I am getting is :

```
*** ERROR ***
A 'ADHeatConductionTimeDerivative' is not a registered object.

If you are trying to find this object in a dynamically loaded library, make sure that
the library can be found either in your "Problem/library_path" parameter or in the
MOOSE_LIBRARY_PATH environment variable.`
```

I seem to be running into the same issue discussed here: https://github.com/idaholab/moose/discussions/24393 - but have not found a good solution. 

I've copied my ParameterStudy input file below (parameter_study_v1.i) and copied the attached the physics input file as well. 

```
[ParameterStudy]
  input = therm_step_elliptical_v8_paramSweep.i
  parameters = 'BCs/radiation_condition/boundary_emissivity'
  quantities_of_interest = 'temp_max/value temp_avg/value'

  sampling_type = lhs
  num_samples = 3
  distributions = 'uniform'
  uniform_lower_bound = 0.2
  uniform_upper_bound = 0.5
  multiapp_mode = batch-reset
[]
```

For further context, I am using Docker and built all of the modules using `make -j 6` in the moose/modules directory. To submit the simulation I am using `../modules/stochastic_tools-opt -i thermal_step_elliptical_v8_paramSweep.i`

Any suggestions would be appreciated! Thanks!

[therm_step_elliptical_v8_paramSweep.txt](https://github.com/idaholab/moose/files/13443424/therm_step_elliptical_v8_paramSweep.txt)

**Comment by GiudGiud** [comment, marked as answer]
Hello

To use the heat transfer (it has been renamed) module you cannot use the stochastic tools executable.
You have to use either:
- the combined module executable
- an application executable that include both heat transfer and stochastic tools

So if you are already building all modules, you should be able to go in moose/modules and build there. A combined executable will be in modules/combined or in modules/ I dont recall

Guillaume

**Reply by cameronmcelfresh** [reply]
Solved using the executable in `moose/modules/combined` 

Thank you!
